composer.json

```diff
@@ -7,7 +7,8 @@
         "symfony/routing": "^3.3",
         "symfony/config": "^3.3",
         "symfony/http-foundation": "^3.3",
-        "erusev/parsedown": "^1.7"
+        "erusev/parsedown": "^1.7",
+        "php-console/php-console": "^3.1"
```

Comment:
perhaps this is just a require-dev? nonetheless, harmless

Reply:
I reverted it later, so it's no worry.